src/redis_cmd.cc

```diff
@@ -318,6 +380,48 @@ class CommandGet : public Commander {
   }
 };
 
+class CommandGetEx : public Commander {
+public:
+    Status Parse(const std::vector<std::string> &args) override {
+        if (std::find_if(args.begin(), args.end(), [](const std::string& str) -> bool {
```

Comment:
I think it is more like a workaround rather than a good solution, as you first check whether there are non-exist options and then use  ParseTtlHelper which is designed for SET.

Reply:
And I think this function is hard to maintain. You can imagine that we need to add a new parameter for the command.

Reply:
I think `PERSIST` is a special case, because it is mutually exclusive with the other ttl arguments, in addition, other cases are the intersection of `Set` and `Getex` commands, the parsing of parameters such as ex, px, etc.

> I think it is more like a workaround rather than a good solution, as you first check whether there are non-exist options and then use ParseTtlHelper which is designed for SET.

Reply:
> And I think this function is hard to maintain. You can imagine that we need to add a new parameter for SET, then we need to also modify GetEX, which is weird.

Yeah, need to think about how to make it more versatile

Reply:
@PragmaTwice  Hi, I thought of adding a parameter to decouple the redis command and the parse function, I think it is more maintainable than before, do you have any ideas about that